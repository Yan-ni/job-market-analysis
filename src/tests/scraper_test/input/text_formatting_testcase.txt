<div>
  <div>
    <p>paragraph <b>test</b> One</p>
    <p>paragraph <strong>test</strong> Two</p>
    <p>paragraph <i>test</i> Three</p>
    <p>paragraph <em>test</em> Four</p>
    <p>paragraph <mark>test</mark> Five</p>
    <p>paragraph <small>test</small> Six</p>
    <p>paragraph <del>test</del> Seven</p>
    <p>paragraph <ins>test</ins> Eight</p>
    <p>paragraph <sub>test</sub> Nine</p>
    <p>paragraph <sup>test</sup> Ten</p>
  </div>
</div>