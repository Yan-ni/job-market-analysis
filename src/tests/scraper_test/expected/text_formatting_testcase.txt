paragraph test One
paragraph test Two
paragraph test Three
paragraph test Four
paragraph test Five
paragraph test Six
paragraph test Seven
paragraph test Eight
paragraph test Nine
paragraph test Ten